submissions/ses3332002/friends-app/app.js

```diff
@@ -40,9 +47,9 @@ function render() {
     let card = document.createElement("div");
     function sexString(age) {
```

Comment:
you really need declare function `sexString` for every `render` function call?
Please, move it go a global scope.

Reply:
Ok, but I use it only in render function.
Explain me, please, when which rule to apply?
Some language guides say that variables and functions should be defined where they are used (as class methods are defined in a class), and that this allows for "modularity" and the ability to reuse entire pieces of code as ready-made modules for other projects, for example: in the eventListeners callback describe all the variables and functions that are used in this callback. This does not "trash" the global scope, but this leads to the fact that these variables and functions are redefined every time each new callback instance is called, and this is obviously bad.
What is interesting: no matter what strategy I use for the location of such "accessory" functions and variables, the reviewers ask to change the approach to the opposite.
So, tell me - when is it necessary to "take out", and when is it not necessary to "take out"?

Reply:
Write me in tg `@andrew_fedin`

Reply:
Ok

Reply:
I moved sexString function to a global scope.